DataSetExplorer/DataSetSerializer/SingleAnnotatorSanityCheckExporter.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using CodeModel.CaDETModel.CodeItems;
+using DataSetExplorer.DataSetBuilder.Model;
+using OfficeOpenXml;
+
+namespace DataSetExplorer.DataSetSerializer
+{
+    class SingleAnnotatorSanityCheckExporter
+    {
+        private readonly string _templatePath = "../../../DataSetSerializer/Template/Single_Annotator_Sanity_Check_Template.xlsx";
+        private readonly string _exportPath;
+        private ExcelPackage _excelFile;
+        private ExcelWorksheet _sheet;
+
+        public SingleAnnotatorSanityCheckExporter(string exportPath)
```

Comment:
This class shares most of the code with the previous one. Can this be a single class that offers two public methods?